server/api/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/shopify', (req, res) => {
```

Comment:
Couple of things here:
Firstly, the webhook will POST to the endpoint we provide so this needs changing to `router.post`
Secondly, let's give this route a more descriptive name. How about `/transaction`? So when the webhook calls the node app Shopify will be POSTing a transaction - makes sense!

Reply:
Can u check now please !